src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,5 +61,14 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
+    
+    /**
+     * returns the name as a printable String
+     */
+	@Override
+	public String getPrintableString() {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.